integrations/optimizely/test/index.test.js

```diff
@@ -1051,37 +772,36 @@ describe('Optimizely', function() {
         window.optimizely.newMockData[2542102702].isActive = false;
         analytics.initialize();
         executeAsyncTest(done, function() {
-          analytics.didNotCall(analytics.track);
+          sinon.assert.notCalled(analytics.track);
         });
       });
     });
   });
 
-  describe('after loading', function() {
-    beforeEach(function(done) {
-      analytics.once('ready', done);
```

Comment:
I don't know `analytics.once('ready', callback);` does or whether it's relevant here. I removed it because it seemed to cause the test suite to hang and skip the tests in this `describe`; though maybe I'm wrong, maybe I was just misreading the test output